.pnp.loader.mjs

```diff
@@ -0,0 +1,249 @@
+import { URL, fileURLToPath, pathToFileURL } from 'url';
```

Comment:
Forgive my ignorance, but what is this code used for?

Reply:
Looks like this was a change from the latest `yarn` version upgrade. It is generated automatically when `yarn` detects that it may be needed, but it's still experimental. We can disable this by `pnpEnableEsmLoader: false` in the `.yarnrc.yml`: https://yarnpkg.com/configuration/yarnrc/#pnpEnableEsmLoader

Reply:
Here's an interesting article on [ESM Loaders](https://ar.al/2021/05/27/make-anything-a-javascript-module-using-node.js-esm-module-loaders/)

Reply:
I ended up removing it since it's unnecessary